service/src/main/java/com/github/szsalyi/flashcard/service/mapper/mappers/UserMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.github.szsalyi.flashcard.service.mapper.mappers;
+
+import com.github.szsalyi.flashcard.entity.UserEntity;
+import com.github.szsalyi.flashcard.service.mapper.common.Mapper;
+import com.github.szsalyi.flashcard.service.user.UserVO;
+import org.dozer.DozerBeanMapper;
+
+/**
+ * Created by ssalyi on 9/17/2017.
+ */
+public final class UserMapper {
```

Comment:
Don't throw out the Spring `ConversionService` approach to type conversions. Instead of creating your own mappers, implement Spring `Converter`s, which in turn will use our registered `DozerBeanMapper` to handle the actual conversion logic.